src/components/OptionRow.tsx

```diff
@@ -288,15 +281,15 @@ function OptionRow(props) {
                                 )}
                             </View>
                         </View>
-                        {Boolean(props.option.customIcon) && (
+                        {!!option.customIcon && (
                             <View
                                 style={[styles.flexRow, styles.alignItemsCenter]}
                                 accessible={false}
                             >
                                 <View>
                                     <Icon
-                                        src={lodashGet(props.option, 'customIcon.src', '')}
-                                        fill={lodashGet(props.option, 'customIcon.color')}
+                                        src={option.customIcon.src}
+                                        fill={option.customIcon.color}
```

Comment:
```suggestion
                                        src={option.customIcon?.src}
                                        fill={option.customIcon?.color}
```

Reply:
@blazejkustra It's checked in [this line](https://github.com/Expensify/App/pull/33077/files/2def510e3683e5197cc3e75f657b6d3ff5596f8d#diff-e7268ea8328cb44bb4424e15bdd5559fb0655b1b83eb1f5a88ac4a2683966d14R284)